broadinstitute/orsp-pub
# Show Data Use Letter form to not authenticated users

## Addresses
There is not Jira ticket
## Changes
Include a high level summary of the changes and what they are intended for.

## Testing
To test this:
As a logged user, go to a Consent Group
In Documents tab, click on Add Document
In the Modal, select Data Use Letter type
Click on "Get shareable link"
Paste the link in an incognito window, or on other browser
Expected Result:
You should see the DUL form and be able to complete and submit it without being logged in the application.
